common/src/main/java/com/microsoft/identity/common/adal/internal/cache/StorageHelper.java

```diff
@@ -715,6 +715,29 @@ private synchronized KeyPair generateKeyPairFromAndroidKeyStore()
             final Locale currentLocale = Locale.getDefault();
             applyKeyStoreLocaleWorkarounds(currentLocale);
 
+            /*
```

Comment:
Hm. This does not seem to be ported to common4j (AndroidWrappedKeyLoader, line 249)

StorageHelper is already marked as deprecated and won't be used in 4j.

Reply:
Lets do that in a separate change